yarn.lock

```diff
@@ -2272,13 +2272,28 @@
     "@formatjs/intl-localematcher" "0.2.32"
     tslib "^2.4.0"
 
+"@formatjs/ecma402-abstract@1.17.0":
```

Comment:
`yarn.lock` changes are probably due to installing updated versions of the deps that were used by `@kaizen/select`.

Given the tests all still pass, I assume they weren't breaking.